src/config_settings.c

```diff
@@ -102,8 +102,9 @@ void setup_default_settings(void)
           {KC_PGUP, KMod_SHIFT},             // Gkey_TiltUp
           {KC_PGDOWN, KMod_SHIFT},           // Gkey_TiltDown
           {KC_INSERT, KMod_SHIFT},           // Gkey_TiltReset
-          {KC_X, KMod_NONE},                // Gkey_Ascend
-          {KC_Z, KMod_NONE},                // Gkey_Descend
+          {KC_X, KMod_NONE},                 // Gkey_Ascend
+          {KC_Z, KMod_NONE},                 // Gkey_Descend
+          {KC_LSHIFT, KMod_NONE},            // Gkey_InstantChargeUp
```

Comment:
WHen I suggested using a Gkey, I meant reusing one of the ones here, for example use Gkey_SpeedMod for instant charge up. What you now did is add a key to the settings list, without adding the option to configure it in the define keys menu.